go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/gin-gonic/gin v1.9.1
 	github.com/golang/protobuf v1.5.4
 	github.com/hashicorp/go-multierror v1.1.1
+	github.com/jedib0t/go-pretty/v6 v6.5.9
```

Comment:
Is this diligenced to be a good choice to depend on?

Reply:
This is a popular golang OSS library for pretty table formatting. The github repo has security scanning enabled and bug bounty for versions >= 6.0.0 https://github.com/jedib0t/go-pretty/security